build.sh

```diff
@@ -5,14 +5,22 @@ set -eux
 rm -rf dist || true
 mkdir -p dist/course/
 
-poetry run python -m translations.generate_po_file
-poetry run python -m core.generate_static_files
+if [[ -z "${SKIP_TRANSLATIONS:-}" ]]; then
+    poetry run python -m translations.generate_po_file
+    poetry run python -m core.generate_static_files
+else
+    # If SKIP_TRANSLATIONS is set, set a default language for this build
+    if [[ -z "${FUTURECODER_LANGUAGE:-}" ]]; then
+        export FUTURECODER_LANGUAGE="en"
+    fi
+fi
+
```

Comment:
Now that we're outputting the build to `course`, I don't see a need for this change.

Reply:
Alrighty, changed it so that the `npm run pwa` command no longer exercises `build.sh`, and reverted this part of `build.sh`